poetry.lock

```diff
@@ -1,11 +1,35 @@
 [[package]]
-category = "dev"
+category = "main"
```

Comment:
I'm really sure about the changes in this file. But I assume poetry does the right thing?